Lib/sqlite3/test/userfunctions.py

```diff
@@ -2,7 +2,7 @@
 # pysqlite2/test/userfunctions.py: tests for user-defined functions and
 #                                  aggregates.
 #
-# Copyright (C) 2005-2007 Gerhard H�ring <gh@ghaering.de>
+# Copyright (C) 2005-2007 Gerhard Häring <gh@ghaering.de>
```

Comment:
What changed on this line? This diff looks suspicious :-)

Reply:
It changes the `ä` from latin-1 to utf-8. So the change is good :)
(I checked with `git show` locally.)

Reply:
```patch
-# Copyright (C) 2005-2007 Gerhard Häring <gh@ghaering.de>
+# Copyright (C) 2005-2007 Gerhard HÃ¤ring <gh@ghaering.de>
```

Reply:
It looks suspicious indeed :) Non-intentional! Will fix.

Reply:
Aha, it was GitHub! f7f9ef1a2423fe4e31e0f881fdaaf0816baf4d35

Reply:
Can you try to revert it?

Reply:
Sure, will do

Reply:
> It changes the ä from latin-1 to utf-8. So the change is good :)

Except that the file is encoded to latin-1 and starts with "#-*- coding: iso-8859-1 -*-" header.

Reply:
If someone wants to convert this file to UTF-8, #-- coding: iso-8859-1 --" header must be removed.

Reply:
That calls for a separate PR, @vstinner ?

Reply:
BTW, reverted in 46642792956cbba07903ae66ae26e21a4462d16f

Reply:
> That calls for a separate PR, @vstinner ?

Technically, the file is properly encoded and valid. I don't see any **need** to change it. If it's not broken, don't fix it :-)